addon/bootstrap.js

```diff
@@ -51,8 +52,11 @@ const prefObserver = {
   }
 };
 
+let localePref = (prefs.getPrefType(LOCALE_PREF) && prefs.getCharPref(LOCALE_PREF)) || "en-US";
+
 function startup(data, reason) { // eslint-disable-line no-unused-vars
   console.log("bootstrap startup called");
+  Cu.import("chrome://screenshots/content/localizedTooltips.jsm");
```

Comment:
I'm guessing we will instead want to lazy load this module up at the top of the file

```js
XPCOMUtils.defineLazyModuleGetter(this, "localizedTooltips",
    "chrome://screenshots/content/localizedTooltips.jsm");
````